CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## X.X.X
 ### PaymentSheet
+* [Fixed] Fixed an issue with selecting from lists on macOS Catalyst. Note that only macOS 11 or later is supported: We do not recommend releasing a Catalyst app targeting macOS 10.15.
```

Comment:
10.15 (or lower) ?

Reply:
There is no 10.16 (they switched to 11), and there are no Catalyst-supporting macOS versions below 10.15

Reply:
Oops. right.